src/services/stats/stats.class.js

```diff
@@ -412,7 +412,7 @@ exports.Stats = class Stats extends Service {
             Math.round(ligneStats?.cnfsActives * 100 / (ligneStats?.nombreConseillersCoselec)) : 0;
 
           if (ligneStats.conseillerIds.length > 0) {
-            const query = { 'conseiller.$id': { $in: ligneStats.conseillerIds }, 'createdAt': {
+            const query = { 'cra.dateAccompagnement': { $in: ligneStats.conseillerIds }, 'createdAt': {
```

Comment:
const query = { 'conseiller.$id': { $in: ligneStats.conseillerIds }, 'cra.dateAccompagnement': {